lib/job.js

```diff
@@ -14,7 +14,13 @@ var Job = module.exports = function Job(args) {
   delete args.agenda;
 
   // Process args
-  args.priority = parsePriority(args.priority) || 0;
+
+  // allow priority overrides (e.g. `job.now('foo', { priority: 'low' })`)
+  // <https://github.com/agenda/agenda/issues/440#issuecomment-316915803>
+  if (args.data && typeof args.data === 'object' && typeof args.data.priority !== 'undefined')
+    args.priority = parsePriority(args.data.priority) || 0;
+  else
+    args.priority = parsePriority(args.priority) || 0;
```

Comment:
All `if` and `else` should use `{}` we should not be relying on indentation at all.